### PR TITLE
Add `experimental.useAtlasEngine` to schema

### LINKED ISSUE #12302: `experimental.useAtlasEngine` missing from JSON schema
The profile setting, `experimental.useAtlasEngine`, should be added to [doc/cascadia/profiles.schema.json](https://github.com/microsoft/terminal/blob/main/doc/cascadia/profiles.schema.json)

Feature was added here: https://github.com/microsoft/terminal/pull/11623

### PR DESCRIPTION
**Pull request by zadjii-msft**
does what it says on the can.

* [x] closes #12302

### CODE REVIEW

**Review by miniksa — APPROVED** [review]

**Comment by miniksa** [comment]
@lhecker give 'em a second.

**Review by lhecker — APPROVED** [review]

**Comment by DHowett** [comment]
lol that typo is gold

File: doc/cascadia/profiles.schema.json [hunk 1]
```diff
@@ -2059,6 +2059,11 @@
           "description": "Use to set a path to a pixel shader to use with the Terminal. Overrides `experimental.retroTerminalEffect`. This is an experimental feature, and its continued existence is not guaranteed.",
           "type": "string"
         },
+        "experimental.useAtlasEngine": {
+          "description": "Enable using the experimental new renderering engine for this profile. This is an experimental feature, and its continued existence is not guaranteed.",
```
Comment (DHowett) [inline doc/cascadia/profiles.schema.json:2063]:
```suggestion
          "description": "Enable using the experimental new rendering engine for this profile. This is an experimental feature, and its continued existence is not guaranteed.",
```

**Comment by DHowett** [comment]
@msftbot merge this in 1 minute

**Review by carlos-zamora — APPROVED** [review]

**Pull request merged**